applications/cheap_flights/data_manager.py

```diff
@@ -34,26 +34,27 @@ def update_airline_name(airline_id) -> str:
         return airline_id
 
 
-def format_flight_route(flight_routes) -> str:
+def format_flight_route(flight_routes) -> list[str]:
     """ Format the flight route information into a string """
 
     def format_datetime(datetime_str):
-        date = datetime.fromtimestamp(datetime_str)
         """ Format datetime object as string 'dd/mm/yy at 00:00 hours'"""
+        date = datetime.fromtimestamp(datetime_str)
         return date.strftime('%d/%m/%y at %H:%M')
 
-    formatted_text = []
+    formatted_routes = []
     for route in flight_routes:
-        fly_from = route["flyFrom"]
```

Comment:
Personal opinion, all these variables to store a value out of a dict isn't really saving any time or effort if you are only using them once.  I'd just use them directly.

Reply:
I see what you mean, but when you "read" the final string, doesn't make it easy to read

from:
```
        full_route_str.append(
            f"{airline} | "
            f"Flight number: {flight_no} - "
            f"{fly_from}({city_from}) to {fly_to}({city_to}) - "
            f"Departure: {local_departure} - Arrival: {local_arrival}.")
```
to
```
        full_route_str.append(
            f"{update_airline_name(route['airline'])} | "
            f"Flight number: {route['flight_no']} - "
            f"{route['flyFrom']}({route['cityFrom']}) to {route['flyTo']}({route['cityTo']}) - "
            f"Departure: {format_datetime(route['dTime'])} - Arrival: {format_datetime(route['aTime'])}.")
```

Reply:
Yup, that's why personally I would leave the airline and the two dates assigned outside and the rest in the string like I showed.  I think it's a good compromise.